styleguide/source/assets/js/tabs.js

```diff
@@ -90,14 +89,23 @@
        * @param {jQuery Object} $tabObj The element which has the .tab() function attached.
        * @param {string} linkHash
        */
-      function switchTabs($tabObj, linkHash) {
+      function switchTabs($tabObj, link) {
+
+        var linkHash = link.getAttribute("href");
```

Comment:
I think the Drupal format for vars should have `$` in front of them

`CamelCasing
For variables that are not constants or constructors, multi-word variables and functions SHOULD be lowerCamelCased.

The first letter of each variable or function SHOULD be lowercase, and the first letter of subsequent words SHOULD be capitalized. There SHOULD NOT be underscores between the words.

In case a variable contains a jQuery object, the variable MUST start with a dollar sign ($):

var $form = $('#search-block-form');
var $inputs = $form.find('input');
`

https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards#variablesandarrays

Reply:
I updated the function docs, but left the variable name as it's not actually a jQuery object.